server/utils/mojPagination/pageResponse.test.ts

```diff
@@ -26,120 +28,83 @@ describe('pageResponseFrom', () => {
+      const page5Of20Pages = pageResponseFrom<number>(pageRequestFrom(5, 5), numberRange(1, 100))
       expect(page5Of20Pages.totalPages()).toEqual(20)
       const accommodatedRange = page5Of20Pages.pageRange(5, 4)
       expect(accommodatedRange).toEqual(numberRange(1, 10))
     })
     it('range not accommodated in the results, at the beginning of the results with a small set of results', () => {
-      const page5Of20Pages = pageResponseFrom<number>(pageRequestFrom(5, 1, 1), numberRange(1, 20))
+      const page5Of20Pages = pageResponseFrom<number>(pageRequestFrom(5, 1), numberRange(1, 20))
       expect(page5Of20Pages.totalPages()).toEqual(4)
       const accommodatedRange = page5Of20Pages.pageRange(5, 4)
       expect(accommodatedRange).toEqual(numberRange(1, 4))
     })
     it('range not accommodated in the results, at the end of results', () => {
-      const page17Of20Pages = pageResponseFrom<number>(pageRequestFrom(5, 17, 1), numberRange(1, 100))
+      const page17Of20Pages = pageResponseFrom<number>(pageRequestFrom(5, 17), numberRange(1, 100))
       expect(page17Of20Pages.totalPages()).toEqual(20)
       const accommodatedRange = page17Of20Pages.pageRange(5, 4)
       expect(accommodatedRange).toEqual(numberRange(11, 20))
     })
     it('range not accommodated in the results, at the end of results with a small set of results', () => {
-      const page17Of20Pages = pageResponseFrom<number>(pageRequestFrom(5, 3, 1), numberRange(1, 20))
+      const page17Of20Pages = pageResponseFrom<number>(pageRequestFrom(5, 3), numberRange(1, 20))
       expect(page17Of20Pages.totalPages()).toEqual(4)
       const accommodatedRange = page17Of20Pages.pageRange(5, 4)
       expect(accommodatedRange).toEqual(numberRange(1, 4))
     })
   })
-  describe('to', () => {
+  describe('resultsTo', () => {
     it('to, where there are more pages', () => {
-      const page3WithPageSize5And20Pages = pageResponseFrom<number>(pageRequestFrom(5, 3, 1), numberRange(1, 100))
+      const page3WithPageSize5And20Pages = pageResponseFrom<number>(pageRequestFrom(5, 3), numberRange(1, 100))
       expect(page3WithPageSize5And20Pages.resultsTo()).toEqual(15)
     })
     it('to, where this is the last page', () => {
-      const page3WithPageSize5And20Pages = pageResponseFrom<number>(pageRequestFrom(5, 3, 1), numberRange(1, 14))
+      const page3WithPageSize5And20Pages = pageResponseFrom<number>(pageRequestFrom(5, 3), numberRange(1, 14))
       expect(page3WithPageSize5And20Pages.resultsTo()).toEqual(14)
     })
   })
 
-  describe('from', () => {
+  describe('resultsFrom', () => {
     it('from', () => {
-      const page3WithPageSize5And20Pages = pageResponseFrom<number>(pageRequestFrom(5, 3, 1), numberRange(1, 100))
+      const page3WithPageSize5And20Pages = pageResponseFrom<number>(pageRequestFrom(5, 3), numberRange(1, 100))
       expect(page3WithPageSize5And20Pages.resultsFrom()).toEqual(11)
     })
   })
-
-  describe('map', () => {
```

Comment:
map function is no longer in PageResponse

Reply:
Out of interest - any particular reason for moving map out of the pageResponse? or did it just end up being neater?